src/PuuClocks.API/internal/service/gameplay.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"puuclocks/internal/models"
+
+	"github.com/google/uuid"
+)
+
+type Gameplay interface {

```

Comment:
wiesz moze jak lepiej nazwac ten serwis?

Reply:
jak nie masz pomysłu to zostaw tak